index.xml

```diff
@@ -12005,12 +11827,51 @@ interface
                     Try running the following steps:
                     <ol>
                       <li>
+                        Let <var>O</var> be <span class='esvalue'>null</span>.
+                      </li>
+
+                      <li>
+                        If <var>target</var> is not a <a class='dfnref'
+                        href='#dfn-namespace'>namespace</a>, and <var>op</var> is not a <a
+                        class='dfnref' href='#dfn-static-operation'>static operation</a>:
+
+                        <ol>
+                          <li>
+                            If the <span class='esvalue'>this</span> value is <span
+                            class='esvalue'>null</span> or <span class='esvalue'>undefined</span>,
+                            set <var>O</var> to <var>realm</var>'s <a class='dfnref external'
+                            href='https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-global'>global
+                            object</a>. (This will subsequently cause a <span
+                            class='estype'>TypeError</span> in a few steps, if <var>target</var> is
+                            not the appropriate global object.)
+
+                            <!-- https://www.w3.org/Bugs/Public/show_bug.cgi?id=18547#c9 -->
+                          </li>
+
+                          <li>Otherwise, set <var>O</var> to the <span class='esvalue'>this</span>
+                          value.</li>
+
+                          <li>If <var>O</var> is a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a>, then <a class='dfnref'
+                          href='#dfn-perform-a-security-check'>perform a security check</a>,
+                          passing <var>O</var>, <var>realm</var>, <var>id</var>, and "method".</li>
+
+                          <li>If <var>O</var> is not a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a> that implements the
+                          interface <var>target</var>, <a href='#ecmascript-throw' class='dfnref
```

Comment:
Also I think that wording actually doesn't work at all. There is no "the" interface whose interface prototype object is `Navigator.prototype`. Navigator, NavigatorID, and NavigatorConcurrentHardware are all interfaces whose interface prototype object is `Navigator.prototype`. Otherwise the previous step, which says "Otherwise, the property exists solely on the interface’s interface prototype object" would not work at all for NavigatorID.

Maybe the solution here is to use a vague phrase like "the interface (or the interface it's being mixed in to, if the interface is actually a mixin)" and have a bug to define this correctly later? If we do that we probably won't fix this until we actually separate out mixins, but that's probably OK given that the current spec seems broken.